services/keylime_registrar.service

```diff
@@ -3,7 +3,12 @@ Description=The Keylime registrar service
 After=network.target
 
 [Service]
+Group=keylime
```

Comment:
tss?

Reply:
Also, is not the idea to be root an later downgrade?

Reply:
The registrar should not need TPM access.

Reply:
but is tss the group, or keylime?

Reply:
keylime is the group of the user. tss of the TPM ownership.

Reply:
Right, so the agent needs to be `keylime:tss` but the verifier/registrar needs to be `keylime:keylime`

Reply:
@mpeters yeah, the agent starts as `root` and drops to `keylime:tss` and the verfier/registrar can just run as `keylime:keylime`.